keda/values.yaml

```diff
@@ -141,8 +141,6 @@ metricsServer:
     successThreshold: 1
 
 webhooks:
-  # -- Enable admission webhooks (this feature option will be removed in v2.12)
-  enabled: true
```

Comment:
I think that this comment was wrong because after talking about this, webhooks will continue being as optional feature. Could you revert related changes? (and update the packed chart)

Reply:
thanks! The change is reverted and chart with index freshly regenerated